src/completions/Tab.ts

```diff
@@ -139,6 +139,28 @@ export class BufferCompletionSource extends Completions.CompletionSourceFuse {
         }
     }
 
+    /** Return the scoredOption[] result for the tab index startswith n */
+    private TabscoredOptionsStartsWidthN(
+        n: number,
+        options: BufferCompletionOption[]
+    ): Completions.ScoredOption[] {
+        n += 1  // Make sure log(n) is valid
+        const nDigits = Math.floor(Math.log10(n))
```

Comment:
This seems a little overengineered to me. I'd suggest converting the numbers to strings and then filtering by startsWith. Is there a reason you didn't want to take that approach?

Reply:
Just that use math looks more formal to me. :)